.github/workflows/build_wheels.yml

```
@@ -55,7 +55,7 @@ jobs:
           - [macos-15-intel, macosx_x86_64]
           - [macos-15, macosx_arm64]
           - [windows-2022, win_amd64]
-        python-version: ['3.9', '3.10', '3.12', 'pypy3.10']
+        python-version: ['3.9', '3.10', '3.11', '3.12', 'pypy3.10']
```

Please add pypy3.11 wheels too
```suggestion
        python-version: ['3.9', '3.10', '3.11', '3.12', 'pypy3.10', 'pypy3.11']
```